spec/javascripts/components/story/collapsed_story/collapsed_story_spec.js

```diff
@@ -4,37 +4,41 @@ import CollapsedStory from 'components/story/CollapsedStory/index';
 import storyFactory from '../../../support/factories/storyFactory';
 
 describe('<CollapsedStory />', () => {
+  const defaultProps = {
+    story: {},
+    onToggle: sinon.stub()
```

Comment:
Are we using this stub in assertions? If so, we need to re-create it for each test.